api/python/tests/integration/test_packages.py

```diff
@@ -666,6 +672,70 @@ def test_s3_set_dir(self):
 
             list_object_versions_mock.assert_called_with('bucket', 'foo/')
 
+    def test_set_dir_update_policy(self):
+        """Verify building a package with update policy. """
+
+        # Test for local src dir
+        path = pathlib.Path(__file__).parent
+        data_dir = pathlib.Path(path, "data")
```

Comment:
Delete above line and 
```suggestion
        data_dir = pathlib.Path(__file__).parent / "data"
```